src/components/calcite-stepper/calcite-stepper.scss

```diff
@@ -1,23 +1,23 @@
 :host {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  position: relative;
-  justify-content: space-between;
-  align-items: stretch;
-  width: 100%;
-  min-width: 100%;
+  @apply flex 
+    flex-row 
+    flex-wrap 
```

Comment:
Does this need to be set?

Reply:
I am not sure, just copied over what was in the styles. You don't think we need them?

Reply:
probably not - those are the defaults

Reply:
we may not need flex-row - it is the initial value